PROTO_tests/tests/dataframe_test.py

```diff
@@ -76,6 +59,7 @@ def build_ak_append():
         day = ak.array([1, 2])
         amount = ak.array([0.5, 5.1])
         bi = ak.array([2**200 + 6, 2**200 + 7])
+        ui = ak.array([6, 7], dtype=ak.uint64)
```

Comment:
This is great! Would you mind adding `2**63` to this so it's in the more representative range for uint?

Reply:
Updated all uint cases with this.